### PR TITLE
Release/0.3.5

### PR DESCRIPTION
**Pull request by oguzhanunlu**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -3,6 +3,18 @@ variable "name" {
   type        = string
 }
 
+variable "app_version" {
+  description = "Version of transformer kinesis"
+  type        = string
+  default     = "5.8.0"
```
Comment (jbeemster) [inline variables.tf:9]: To confirm that 5.8.0 has no major config changes needed for this module to still work without the config override?

**Review by jbeemster — APPROVED** [review]
One quick query but looks good!  Can you also prepare a CHANGELOG change commit as well please?

File: variables.tf (hunk 1 above)
Comment (oguzhanunlu) [reply variables.tf:9]: yes, it should work without config override, thanks for the heads up!

I'll prepare them now

Comment (oguzhanunlu) [reply variables.tf:9]: done!

**Review by jbeemster — APPROVED** [review]

**Pull request merged**